python/examples/influxdb.py

```diff
@@ -0,0 +1,105 @@
+import os
+import base64
+import secrets
+from influxdb import InfluxDBClient
+import influxdb
+from pshconfig import Config
+import traceback, sys
+
+
+def test_output():
+
+    # Create a new Config object to ease reading the Platform.sh environment variables.
+    # You can alternatively use os.environ yourself.
+    config = Config()
+
+    # Get the credentials to connect to the InfluxDB service.
+    credentials = config.credentials('influxdb')
+
+    try:
+
+        # client = InfluxDBClient(host=credentials['host'], port=credentials['port'], database='deploys')
+
+        user = 'deploy_user'
+        password = 'password'
+        dbname = 'deploys'
+
+        # client.switch_user('root', 'root')
+
+        # client.create_user(username=user, password=password)
+        #
```

Comment:
Vestigial?

Reply:
Ya I'm still trying to figure out Influx, so most of it will go once its up and running.